src/helpers/mapSupplierToContentfulFields.spec.js

```diff
@@ -56,10 +68,10 @@ describe("mapSupplierToContentfulFields", () => {
   });
 
   it("handles empty supplier fields", () => {
-    let supplier = testSupplier;
+    let supplier = structuredClone(testSupplier);
```

Comment:
This test was previously overwriting the `testSupplier` object, which would mean it could cause test failures if any tests are added after it.  Therefore, I've updated it to match the test I added where it creates a clone of the `testSupplier` to ensure it doesn't affect the original object and avoid issues in the future